src/main/java/edu/umich/ctools/sectionsUtilityTool/FriendServlet.java

```diff
@@ -154,11 +154,13 @@ private StringEntity socialLoginApiRequestBody(String spURL, String nonUmichEmai
         apiPostBody.put(Utils.SOCIAL_LOGIN_SERVICE_NAME, "UMICH Invite");
         apiPostBody.put(Utils.SOCIAL_LOGIN_EMAIL_ADDRESS, nonUmichEmail);
         apiPostBody.put(Utils.SOCIAL_LOGIN_EMAIL_SUBJECT, String.format("UMICH Invitation for %s %s" ,friendFirstName, friendLastName));
-        apiPostBody.put(Utils.SOCIAL_LOGIN_SPONSOR_EPPN, Utils.FRIEND_CONTACT_EMAIL);
-        apiPostBody.put(Utils.SOCIAL_LOGIN_SPONSOR_MAIL, Utils.FRIEND_CONTACT_EMAIL);
```

Comment:
If Utils.FRIEND_CONTACT_EMAIL is no longer used, maybe it should be removed from the Utils.java file.

Reply:
I made the social login feature a configurable option so that it can be swtiched with friend service if we are not ready with social login by end of december as there are so many moving elements out side of CCM so `Utils.FRIEND_CONTACT_EMAIL` needs to be there for the moment